pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -63,6 +63,9 @@
 
     protected volatile boolean isFenced;
 
+    // When set to false, this inactive topic can not be deleted
+    protected boolean brokerDeleteInactiveTopicsEnabled;
```

Comment:
I think it's better to named `deleteWhileInactive`.

Reply:
Done.